src/canonmn_int.cpp

```diff
@@ -1854,34 +1854,37 @@ constexpr TagDetails canonCsLensType[] = {{1, "Canon EF 50mm f/1.8"},
                                           {61182, "Canon RF 35mm F1.8 MACRO IS STM"},
-                                          {61182, "Canon RF 50mm F1.2L USM"},                       // 1
-                                          {61182, "Canon RF 24-105mm F4L IS USM"},                  // 2
-                                          {61182, "Canon RF 28-70mm F2L USM"},                      // 3
-                                          {61182, "Canon RF 85mm F1.2L USM"},                       // 4
-                                          {61182, "Canon RF 24-240mm F4-6.3 IS USM"},               // 5
-                                          {61182, "Canon RF 24-70mm F2.8L IS USM"},                 // 6
-                                          {61182, "Canon RF 15-35mm F2.8L IS USM"},                 // 7
-                                          {61182, "Canon RF 50mm F1.8 STM"},                        // 8
-                                          {61182, "Canon RF 70-200mm F4L IS USM"},                  // 9
-                                          {61182, "Canon RF 28-70mm F2L USM"},                      // 10
-                                          {61182, "Canon RF 85mm F2 MACRO IS STM"},                 // 11
-                                          {61182, "Canon RF 100-500mm F4.5-7.1L IS USM + RF1.4x"},  // 12
-                                          {61182, "Canon RF 35mm F1.8 MACRO IS STM"},               // 13
-                                          {61182, "Canon RF 85mm F1.2L USM DS"},                    // 14
-                                          {61182, "Canon RF 24-70mm F2.8L IS USM"},                 // 15
-                                          {61182, "Canon RF 15-35mm F2.8L IS USM"},                 // 16
-                                          {61182, "Canon RF 24-240mm F4-6.3 IS USM"},               // 17
-                                          {61182, "Canon RF 70-200mm F2.8L IS USM"},                // 18
-                                          {61182, "Canon RF 600mm F11 IS STM"},                     // 19
-                                          {61182, "Canon RF 600mm F11 IS STM + RF1.4x"},            // 20
-                                          {61182, "Canon RF 600mm F11 IS STM + RF2x"},              // 21
-                                          {61182, "Canon RF 800mm F11 IS STM"},                     // 22
-                                          {61182, "Canon RF 800mm F11 IS STM + RF1.4x"},            // 23
-                                          {61182, "Canon RF 800mm F11 IS STM + RF2x"},              // 24
-                                          {61182, "Canon RF 24-105mm F4-7.1 IS STM"},               // 25
-                                          {61182, "Canon RF 100-500mm F4.5-7.1L IS USM"},           // 26
-                                          {61182, "Canon RF 100-500mm F4.5-7.1L IS USM + RF2x"},    // 27
```

Comment:
I'd prefer if you could not remove these comments, it introduces an unnecessary diff.  
And other duplicate values still have them in this array, so let's just leave them and stay consistent.

Reply:
I.e. maintaining the comments for RF lenses and re-sequencing every time a lens is added to match exiftool will introduce unnecessary diffs in the future. Adding in sequence will make syncing w/ `RFLensType` table that has a clear sorting order by ID.

Reply:
Well IMHO the sorting here is not obvious because the RF ID doesn't exist in this list as it doesn't have any meaning. In the CsType list they all have only one ID (61182).     
Thus, it's only a matter of time until someone adds a lens to this table in what you would call the "wrong order", which is why I'd just from the start not worry about the order and ignore it. As I said, the order is meaningless here anyway.   

If you want to make this nicer, I'd argue it's better to have the value of `61182` directly point to the `canonRFLensType` list which would avoid the duplication.

Reply:
exiftool keeps two lists and the RF lenses always sorted in the ID order in both (don't know if it's manual or by a script). We could simply put a comment above the first 61182 lens: "please keep in sync w/ sorted RFLensType list below"

> If you want to make this nicer, I'd argue it's better to have the value of `61182` directly point to the `canonRFLensType` list which would avoid the duplication.

Yes, please, but it's not obvious to me how to do this ATM.

Reply:
adding `"please keep in sync w/ sorted RFLensType list below"` would be a compromise I can live with :wink:

Reply:
Will do, just have to get around to it...

Reply:
Done